ui/apps/platform/src/Containers/VulnMgmt/List/Cves/CveBulkActionDialogue.js

```diff
@@ -157,22 +157,18 @@ const CveBulkActionDialogue = ({ closeAction, bulkActionCveIds, cveType }) => {
         policyQueryOptions
     );
 
-    if (
-        !policyLoading &&
-        policyData &&
-        policyData.results &&
-        policyData.results.length &&
-        policies.length === 0
-    ) {
-        const existingPolicies = policyData.results
-            .filter((policyToFilter) => !policyToFilter?.isDefault)
-            .map((policyToMap, idx) => ({
-                ...policyToMap,
-                value: idx,
-                label: policyToMap.name,
-            }));
-        setPolicies(existingPolicies);
-    }
+    useEffect(() => {
+        if (!policyLoading && policyData?.results?.length) {
+            const existingPolicies = policyData.results
+                .filter((policyToFilter) => !policyToFilter?.isDefault)
+                .map((policyToMap, idx) => ({
+                    ...policyToMap,
+                    value: idx,
+                    label: policyToMap.name,
+                }));
+            setPolicies(existingPolicies);
+        }
+    }, [policyLoading, policyData]);
```

Comment:
I'm curious if the `useEffect` is necessary here. Wouldn't it go through this code block again when `policyLoading` or ``policyData` changes anyway?